examples/external-model/samples.json

```diff
@@ -0,0 +1,7 @@
+{
+  "samples": [
+    {
+      "embedding_input": [1, 23, 3, 4, 5, 234, 45]
```

Comment:
Can you switch to iris, since the input can be the raw features? You should be able to use the model from my blog post (https://s3-us-west-2.amazonaws.com/cortex-blog/tf-serving-k8s/iris.tar.gz), just download it and re-zip it and upload it and it should work. The `samples.json` data is also in the post (https://blog.cortex.dev/tensorflow-serving-kubernetes/)